src/app/components/Article/Article.module.scss

```diff
@@ -39,7 +39,7 @@
     color: $color-700-neutral;
   }
 
-  &__readMore {
+  .readMore {
```

Comment:
`&__readMore` because when you put it this way .readMore inside .article you increase specificity for no reason. It’s better to keep it as low as possible